MoreLinq/Fold.g.tt

```diff
@@ -37,8 +37,8 @@ namespace MoreLinq
                 Count         = i,
                 CountElements = istr + " " + (i == 1 ? "element" : "elements"),
                 CountArg      = istr,
-                FolderArgs    = "folder" + istr + ": folder",
-
+                Elements      = string.Join(", ", Enumerable.Range(0, i)
+                    .Select(j => FormattableString.Invariant($"elements[{j}]"))),
```

Comment:
Please fix the code formatting by aligning the chained methods:

```suggestion
                Elements      = string.Join(", ", Enumerable.Range(0, i)
                                                            .Select(j => FormattableString.Invariant($"elements[{j}]"))),
```

And, as a nit, preferably using the query syntax:

```suggestion
                Elements      = string.Join(", ", from j in Enumerable.Range(0, i)
                                                  select FormattableString.Invariant($"elements[{j}]")),
```

Also, since this is only ever used once, I would even suggest to in-line the expression.

---
> Addressed, using `FormattableString.Invariant` (cleaner)

PS Just saying “cleaner” doesn't offer much, but in the absence of any technical/objective argument, what it is saying is that the [original suggestion](https://github.com/morelinq/MoreLINQ/pull/961#discussion_r1096072797) was somehow quick and dirty. I'd either entirely refrain or qualify with a technical/objective argument for the reviewers and future readers/maintainers.

Reply:
Fixed with 9d6b44c929eb536c36aa18ae5b0373f92c0a125f and 22edea258682c77f4cc4af470dfd93be088038cd.